doc/mkdocs/Makefile

```diff
@@ -1,21 +1,21 @@
 # serve the site locally
-serve: prepare_files
+serve: prepare_files style_check
 	venv/bin/mkdocs serve
 
-serve_dirty: prepare_files
+serve_dirty: prepare_files style_check
 	venv/bin/mkdocs serve --dirtyreload
 
-build: prepare_files
+build: prepare_files style_check
 	venv/bin/mkdocs build
 
-# create files that are not versioned inside the mkdocs folder
+# create files that are not versioned inside the mkdocs folder (images, examples)
 prepare_files: clean
-	# create subfolders
 	mkdir docs/examples
-	# copy images
-	cp -vr ../json.gif docs/images
-	# copy examples
-	cp -vr ../examples/*.cpp ../examples/*.output docs/examples
+	cp -r ../json.gif docs/images
```

Comment:
This file (or the whole folder) is not in `.gitignore`, the others are.

Reply:
Good point! Fixed.